codespell_lib/data/dictionary.txt

```diff
@@ -28015,6 +28032,8 @@ semding->sending
 sement->cement, segment,
 sementation->segmentation
 semented->cemented, segmented,
+sementic->semantic
+sementics->semantics
```

Comment:
If you don't mind adding
```text
sementically->semantically
```
https://github.com/search?q=sementically&type=code

Reply:
[bc58273](https://github.com/codespell-project/codespell/pull/2370/commits/bc5827378dc7f488c617b9163ed1940d76a65bbd)